DokuwikiPage.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * This is the Dokuwiki export for FINDOLOGIC.
+ *
+ * If any bugs occur, please submit a new issue
+ * @see https://github.com/findologic/dokuwiki-plugin-findologic-xml-export/issues/new
+ * @author Dominik Brader <support@findologic.com>
+ */
+
+class DokuwikiPage
+{
+    /**
+     * This is the time format as it gets outputted to the template.
+     * DAY.MONTH YEAR (HOUR:MINUTE)
+     */
+    const TIME_FORMAT = '%d.%B %Y (%H:%M)';
```

Comment:
Format in template, use `localizeddate` Twig extension.

Reply:
Note4Me: should be done locally